docs/strictdoc_01_user_guide.sdoc

```diff
@@ -435,25 +606,29 @@ comments can be single-line or multiline.
     [REQUIREMENT]
     UID: REQ-001
     STATEMENT: StrictDoc shall enable requirements management.
-    COMMENT: Clarify the meaning or give additional information here.
-    COMMENT: >>>
-    This is a multiline comment.
+[/FREETEXT]
 
-    The content is split via \n\n.
+[/SECTION]
 
-    Each line is rendered as a separate paragraph.
-    <<<
+[SECTION]
+UID: REQUIREMENT_FIELD_STATEMENT
+TITLE: Statement
+
+[FREETEXT]
+The statement of the requirement. The field can be single-line or multiline.
+At least the ``STATEMENT`` field should be present.
```

Comment:
I re-used the text from above (line 459)
```
Currently, all ``[REQUIREMENT]``'s fields are optional but most of the time at
least the ``STATEMENT`` field as well as the ``TITLE`` field should be
present.
```
I agree that a Requirement always SHALL have a Statement, i.e. the user should have a very very good reason not to include a Statement field. A Requirement without Statement is either WorkInProgress or just some informational freetext.
For some (historical?) reasons it is currently optional in the ``[REQUIREMENT]``'s field definitions. The same is true for the ``UID``/``TITLE`` field.

Maybe we should change the text in shall/must for the ``STATEMENT`` and ``UID``/``TITLE`` fields, even when the internal code handles it as optional.

Reply:
> I re-used the text from above (line 459)

Yes, there it makes sense within the surrounding context. Taken out, it is less clear IMO.

The historical reasons were the following: it is very often that you create requirements quickly one after another before knowing or filling out exactly all the details - so it should be possible to create a requirement with either a title only or a statement only. I think it is still an acceptable behavior. Let's not start discussing this change within this relatively trivial documentation update PR, otherwise we will grow out of scope.